compiler/env/OMRKnownObjectTable.hpp

```diff
@@ -87,6 +87,10 @@ class OMR_EXTENSIBLE KnownObjectTable
    // API for checking if an known object is an array with immutable elements
    bool isArrayWithConstantElements(Index index);
 
+   // API for checking if an known object is an array whose elements are immutable
+   // unless they are zero
+   virtual bool isArrayWithStableElements(Index index);
```

Comment:
This will be overridden statically through the extensible class mechanism, so it shouldn't be virtual

Reply:
fixed